cyto_dl/train.py

```diff
@@ -93,6 +100,10 @@ def train(cfg: DictConfig, data=None) -> Tuple[dict, dict]:
         log.info("Logging hyperparameters!")
         utils.log_hyperparameters(object_dict)
 
+    if use_batch_tuner:
```

Comment:
This seems related to the new block up starting on line 62 - could it go up there for readability?  Could they also be a single condition (it seems they are logically bound?).

Reply:
Good point. I have it split up because the data arguments need to be changed if the batch size tuner is used, but then the actual batch tuning requires an initialized datamodule. I can rearrange things to make them a little closer, but that might be purely aesthetic as they can't be fully grouped.